app/src/main/java/com/sdp/swiftwallet/common/FirebaseAuthModule.java

```diff
@@ -1,17 +1,31 @@
 package com.sdp.swiftwallet.common;
 import com.google.firebase.auth.FirebaseAuth;
 
-import dagger.Binds;
+
 import dagger.Module;
 import dagger.Provides;
 import dagger.hilt.InstallIn;
-import dagger.hilt.android.components.ActivityComponent;
+import dagger.hilt.components.SingletonComponent;
+import javax.inject.Inject;
+import javax.inject.Singleton;
 
 @Module
-@InstallIn(ActivityComponent.class)
+@InstallIn(SingletonComponent.class)
 public class FirebaseAuthModule {
+
+    /**
+     * For hilt
+     */
+    @Inject
+    public FirebaseAuthModule(){ }
```

Comment:
This has no use here, I tested it on my side but don't touch it for now I did some stuff on another branch to transfer firebaseUtil directly into the hilt module, it makes more sense. So don't bother to change it, I already did so I can just make a small PR after.

Reply:
Not my code once again